src/client/Microsoft.Identity.Client/Internal/Requests/Silent/CacheSilentStrategy.cs

```diff
@@ -221,24 +221,6 @@ private async Task<MsalTokenResponse> TryGetTokenUsingFociAsync(CancellationToke
             return null;
         }
 
-        private async Task<MsalTokenResponse> RefreshAccessTokenAsync(MsalRefreshTokenCacheItem msalRefreshTokenItem, CancellationToken cancellationToken)
```

Comment:
i don't know why this class was changed. then, you don't need to pass a bunch of parameters around.

Reply:
This was done to remove code duplication as the silent request and the OBO request will use very similar code going forward. I agree that passing additional variables around is not ideal but it might be better than duplicate code. @neha-bhargava do you think you can figure out a way to reduce the number of parameters being passed around. If not then I guess we can take it as is.

Reply:
The methods are moved from here to the [SilentRequestHelper.cs](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/92ed27f24009f1b1574d4b5fdc5522be4819bfc8/src/client/Microsoft.Identity.Client/Internal/Requests/SilentRequestHelper.cs#L13). These methods were also used for OBO refresh flow so to reduce duplicate code I moved them.

Reply:
There are more params because the requests use class inheritance and some common code is in the base class. One of the reasons I don't like class inheritance.